libbeat/paths/paths.go

```diff
@@ -28,6 +28,8 @@
 //
 // path.config - Configuration files and Elasticsearch template default location
 //
+// system.hostfs - supplies an alternate filesystem root for containerized environments
```

Comment:
Wdyt about calling it `path.hostfs` now that it is another configurable path, and not only used by the system module?

Reply:
Yah, it might be worth it to have a second one for backwards compatibility?

Reply:
Yep, we would keep both at least till 8.0.